MinVer.Lib/RepositoryExtensions.cs

```diff
@@ -2,13 +2,12 @@ namespace MinVer.Lib
 {
     using System.Collections.Generic;
     using System.Linq;
-    using LibGit2Sharp;
 
     internal static class RepositoryExtensions

```

Comment:
Why keep this extensions class when the `Repository` class it's extending is now in MinVer.LIb and not from LibGit2Sharp?

Reply:
Good question. And in fact, it would be good to get rid of it, since then `Repository` would not need any public properties. Good spot, thanks.